plugin/trino-accumulo/src/test/java/io/trino/plugin/accumulo/TestAccumuloConnectorTest.java

```diff
@@ -72,22 +81,16 @@ protected TestTable createTableWithDefaultColumns()
         throw new SkipException("Accumulo connector does not support column default values");
     }
 
-    @Override
-    public void testCreateSchema()
-    {
-        // schema creation is not supported
-    }
-
     @Override
     public void testAddColumn()
     {
-        // Adding columns via SQL are not supported until adding columns with comments are supported
+        throw new SkipException("Accumulo connector does not support adding columns");
```

Comment:
Sorry for missing this earlier but can you please add a negative assertion before the SkipException? Same for testDropColumn (message is `This connector does not support dropping columns`)?

```java
assertThatThrownBy(() -> super.testAddColumn()).hasMessageMatching("This connector does not support adding columns");
```

For reference the exception we get is:

```
java.lang.RuntimeException: This connector does not support adding columns
...
Suppressed: java.lang.Exception: SQL: ALTER TABLE test_add_column_7nlod ADD COLUMN a varchar
		at io.trino.testing.DistributedQueryRunner.execute(DistributedQueryRunner.java:479)
		... 29 more
Caused by: io.trino.spi.TrinoException: This connector does not support adding columns
...
```

Reply:
Good point. Thank you @hashhar